### PR TITLE
Base es-button component

### PR DESCRIPTION
**Pull request by pbishop16**
Addresses #4

### CODE REVIEW

**Comment by pbishop16** [comment]
@MelSumner My apologies for the delay. Please let me know what needs refinement or what additional tests you would like.

File: addon/components/es-button.js [hunk 1]
```diff
@@ -1,21 +1,76 @@
...
+      'type'
+    ],
+
+    //attributes
+    dataRole: null,
+    label: null,
+    type: null,
+
+    //sizes
+    buttonBlock: false,
```
Comment (MelSumner) [inline addon/components/es-button.js]: how about "isBlock" or some other verb in front for booleans?

Comment (pbishop16) [reply addon/components/es-button.js]: @MelSumner Ah will do. I had that but changed it.

**Comment by MelSumner** [comment]
Approved- thank you @pbishop16 !

**Comment by pbishop16** [comment]
@rwwagner90 @MelSumner Greatly appreciated!!!

**Pull request merged**